### ISSUE TITLE
Fix neuprint_ROI_connectivity

### ISSUE THREAD

**Issue by jefferis** [natverse/neuprintr#87]
I can't seem to get it to work at all when `cached=FALSE` and when `cached=TRUE` it also fails if there are no connections between some of the requested ROIs. 

```
aba <- neuprint_ROI_connectivity(neuprint_ROIs(superLevel = TRUE), cached=TRUE)
```

**Comment by jefferis** [comment]
(see earlier issues in #20)

**Comment by usenatverse** [comment]
Closed by #88
